html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <div>
+        <label for="email">Email</label><br />
+        <input type="email" /><br />
```

Comment:
Your `label` here won't work. You're missing something on your `input` that connects your `label` to the `input`.